_api-reference/index-apis/create-index.md

```diff
@@ -94,6 +91,7 @@ Setting | Description
 :--- | :---
 index.number_of_replicas | The number of replica shards each primary shard should have. For example, if you have 4 primary shards and set `index.number_of_replicas` to 3, the index has 12 replica shards. Default is 1.
 index.auto_expand_replicas | Whether the cluster should automatically add replica shards based on the number of data nodes. Specify a lower bound and upper limit (for example, 0-9), or `all` for the upper limit. For example, if you have 5 data nodes and set `index.auto_expand_replicas` to 0-3, then the cluster does not automatically add another replica shard. However, if you set this value to `0-all` and add 2 more nodes for a total of 7, the cluster will expand to now have 6 replica shards. Default is disabled.
+index.codec |  Determines how the index’s stored fields are compressed and stored on the disk. This setting impacts the size of the index shards and the performance of the operations on the index. Available values are: `default', 'best_compression`, `zstd` and `zstd_no_dict`. Two new codecs are introduced in OpenSearch 2.9: `zstd` and `zstd_no_dict`. They provide an option to configure the compression level as an index setting `index.codec.compression_level` which is not available for other codecs. For information about each setting, see [Index codec settings](#Index-codec-settings).
```

Comment:
Can we add a line over here that index.codec settings cannot be updated in real-time.
Similarly, should we add index.codec.compression_level also over here in dynamic settings, with its information? Also index.codec.compression_level can also not be updated at real time.